### ISSUE TITLE
Question including "xxx"

### ISSUE THREAD

**Issue by pabloavegno** [dipendrapkrl/TriviaHelper#32]
Hello, I am a new user. I did no read any post or setting referring to the phrases between " ". I think that this part in the question have to be literally in the googling process without deleting the comoun words like "the" and the other words named to be bypassed. Do you understand my suggestion?

**Comment by dipendrapkrl** [comment]
Nope unfortunately. could you rephrase it please.

**Comment by pabloavegno** [comment]
I would  like to send an image, but it will try to explain with words. Imagine that there is a question which says ... Which of this actors is present in "the cat in the oven" film. If you put "the" word as a word no to look for, you will break the name of the film while searching. So, I suggest not to take out any word from phrases between " ". Now is clear what I mean?

**Comment by Raj4everr** [comment]
Let me try explaining what you mean...whatever common words or letters coming specially within double quotes " " should not be omitted during answer search...like in the above example"the cat in the oven"...wherein 'the' and 'in' are pretty common...but will be omitted during search...

**Comment by pabloavegno** [comment]
Correct. And also the " " signs have also not to be ommited.

**Comment by dipendrapkrl** [comment]
yea. Makes sense. That could be considered for future enhancements. I will consider that whenever I have time. Thanks.
